### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by Monty-00**

### CODE REVIEW

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Monty-00 Changes required
1. Rename "images and videos" as "Images and Videos"
2. Images and Videos should contain two sub-folders "Images" and "Videos".
3. File names should be self-explanatory. For ex., you are not allowed to have names like "IMG-20200728-WA0000.jpg".
4. Rename folder "Wall climbing" as "Mechanical Design".
5. Add CAD images(snapshots in .jpg format) of Solidworks files inside Mechanical Design folder.
6. Poster and Report is missing. Create a folder named "Poster and Report" and add one project report and one project preview poster inside that folder.
7. In README.md, add flowchart.
8. In README.md, hyperlink Team Member names and Mentor names with their GitHub profile link.

**Comment by agshubhs** [comment]
> @Monty-00 Changes required
> 
> 1. Rename "images and videos" as "Images and Videos"
> 2. Images and Videos should contain two sub-folders "Images" and "Videos".
> 3. File names should be self-explanatory. For ex., you are not allowed to have names like "IMG-20200728-WA0000.jpg".
> 4. Rename folder "Wall climbing" as "Mechanical Design".
> 5. Add CAD images(snapshots in .jpg format) of Solidworks files inside Mechanical Design folder.
> 6. Poster and Report is missing. Create a folder named "Poster and Report" and add one project report and one project preview poster inside that folder.
> 7. In README.md, add flowchart.
> 8. In README.md, hyperlink Team Member names and Mentor names with their GitHub profile link.

@Monty-00  Do the following changes and tag me or @SanjeevKrishnan once you ar done with this

File: README.md [hunk 1]
```diff
@@ -1,2 +1,62 @@
...
+1. It is connected with wire , which restricts its motion.
+2. It can not go high from a particular height because of pressure difference.
+
+## Future Improvements
+1. It can be made remote controlled.
+2. It can be made wireless with help of devices like Bluetooth.
+3. By improving electronic system and mechanical strength, it can reach greater heights.
+
+## Team Members
+1. Monty (https://github.com/Monty-00)
```
Comment (SanjeevKrishnan) [inline README.md]: The syntax is wrong. It should be `[Monty](https://github.com/Monty-00)`. Change this for all members and mentors

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Monty-00 Changes Required
1. Few file names in Mechanical Design folder are not self-explanatory. For ex., CAD Design 1, CAD Design2,   Make sure that name is matching with its respective solidwork file.
2. Flowchart is missing in README.md

**Review by SanjeevKrishnan — APPROVED** [review]
@Monty-00 All requirements are met! PR looks fine.
@Shubhanshu07 I suggest for merge.

**Comment by agshubhs** [comment]
@Monty-00 Great Work Guys!!

**Comment by agshubhs** [comment]
@Monty-00 Make a new PR.

**Pull request merged**